Compiler warnings in WCSimWCDigitizer.{hh,cc}

Compiling WCSim with gcc/g++ version 4.8.5 reports the warnings related to WCSimWCDigitizer.{hh,cc} in the attached file [make.log](https://github.com/spradlin/WCSim/files/6566128/make.log).
The warnings were generated by `touch`ing the source files WCSimWCDigitizer.{hh,cc} and `make`ing.

The warning
```
src/WCSimWCDigitizer.cc: In member function 'bool WCSimWCDigitizerBase::AddNewDigit(int, int, double, double, std::vector<int>)':
src/WCSimWCDigitizer.cc:116:124: warning: declaration of 'peSmeared' shadows a member of 'this' [-Wshadow]
 bool WCSimWCDigitizerBase::AddNewDigit(int tube, int gate, double digihittime, double peSmeared, std::vector<int> digi_comp)

                                            ^
```
raises questions about the need of data member `WCSimWCDigitizerBase::peSmeared`.  The result of this warning is that it is clearly not used in `WCSimWCDigitizerBase::AddNewDigit()`.  Where is it used?

It is used in `WCSimWCDigitizerSKI::DigitizeHits()`, and only there.  In the current implementation, `WCSimWCDigitizerBase::peSmeared` is never used to pass a value among member functions.  Further, it is reinitialized to 0 several times per event, and does not fit into the concept of a Digitizer base class as i understand it.  It might be useful in a derived class that sought to closely emulate hardware, but that does not appear to be the case with the current implementation of `WCSimWCDigitizerSKI`.

The solution that makes most sense to me is to eliminate the data member `WCSimWCDigitizerBase::peSmeared` and declare a variable in the scope of `WCSimWCDigitizerSKI::DigitizeHits()` to serve its purpose.

--- a/include/WCSimWCDigitizer.hh
+++ b/include/WCSimWCDigitizer.hh
@@ -52,7 +52,6 @@ public:
 protected:
   void ReInitialize() { DigiStoreHitMap.clear(); }
 
-  G4double peSmeared;
 
   WCSimDetectorConstruction* myDetector; ///< Get the geometry information
   WCSimWCDAQMessenger* DAQMessenger;     ///< Get the /DAQ/ .mac options

--- a/src/WCSimWCDigitizer.cc
+++ b/src/WCSimWCDigitizer.cc
@@ -23,8 +23,8 @@ WCSimWCDigitizerBase::WCSimWCDigitizerBase(G4String name,
 										   WCSimDetectorConstruction* inDetector,
 										   WCSimWCDAQMessenger* myMessenger,
 										   DigitizerType_t digitype,
-										   G4String detectorElement)
-  :G4VDigitizerModule(name), myDetector(inDetector), DAQMessenger(myMessenger), DigitizerType(digitype),DigitizerClassName(""), detectorElement(detectorElement)
+										   G4String detElem)
+  :G4VDigitizerModule(name), myDetector(inDetector), DAQMessenger(myMessenger), DigitizerClassName(""),DigitizerType(digitype), detectorElement(detElem)
 {
   // G4String colName = "WCDigitizedStoreCollection";
   G4String colName;
@@ -186,10 +186,10 @@ void WCSimWCDigitizerBase::SaveOptionsToOutput(WCSimRootOptions * wcopt)
 // *******************************************
 
 WCSimWCDigitizerSKI::WCSimWCDigitizerSKI(G4String name,
-                                         WCSimDetectorConstruction* myDetector,
+                                         WCSimDetectorConstruction* inDetector,
                                          WCSimWCDAQMessenger* myMessenger,
-                                         G4String detectorElement)
-  : WCSimWCDigitizerBase(name, myDetector, myMessenger, kDigitizerSKI, detectorElement)
+                                         G4String detElem)
+  : WCSimWCDigitizerBase(name, inDetector, myMessenger, kDigitizerSKI, detElem)
 {
   DigitizerClassName = "SKI";
   GetVariables();
@@ -247,6 +247,7 @@ void WCSimWCDigitizerSKI::DigitizeHits(WCSimWCDigitsCollection* WCHCPMT) {
       double intgr_start=0;
       double upperlimit=0;
       G4double efficiency = 0.985; // with skrn1pe (AP tuning) & 30% QE increase in stacking action
+      G4double peSmeared;
 
       // Variables to store photon uniqueid that make up a digit
       int digi_unique_id   = 0;

